### PR TITLE
Add: MITMediaLab.Scratch.3 version 3.24.0

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21995)

### CODE REVIEW

**Comment by quhxl** [comment]
Different scopes do not seem possible.

File: manifests/m/MITMediaLab/Scratch/3/3.24.0/MITMediaLab.Scratch.3.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://downloads.scratch.mit.edu/desktop/Scratch%203.24.0%20Setup.exe
+  InstallerSha256: 985F33F2B87C059CF05C3517DFCB0905B1A8385FC327C82FB1282AD22486B191
+  UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/m/MITMediaLab/Scratch/3/3.24.0/MITMediaLab.Scratch.3.installer.yaml:17]:
```suggestion
  InstallerSwitches:
    Custom: /CURRENTUSER
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://downloads.scratch.mit.edu/desktop/Scratch%203.24.0%20Setup.exe
  InstallerSha256: 985F33F2B87C059CF05C3517DFCB0905B1A8385FC327C82FB1282AD22486B191
  InstallerSwitches:
    Custom: /ALLUSERS
  UpgradeBehavior: install
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/m/MITMediaLab/Scratch/3/3.24.0/MITMediaLab.Scratch.3.installer.yaml (hunk 1 above)
Comment (quhxl) [reply manifests/m/MITMediaLab/Scratch/3/3.24.0/MITMediaLab.Scratch.3.installer.yaml:17]: Are you sure that works? I tried it and it didn't...

Comment (OfficialEsco) [reply manifests/m/MITMediaLab/Scratch/3/3.24.0/MITMediaLab.Scratch.3.installer.yaml:17]: Requires Administrator
image

Does not require Administrator
image

After testing:
The issue with the Machine one is that it requires Powershell to be run as Administrator
`.\Scratch3.24.0Setup.exe /S /ALLUSERS` without admin = nothing
`.\Scratch3.24.0Setup.exe /S /ALLUSERS` as admin = installs

**Comment by OfficialEsco** [comment]
The bot is having a bad day/week, hopefully they fix it before the weekend

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**